### PR TITLE
Update eips impls

### PR DESCRIPTION
**Pull request by pdobacz**
Just a cosmetic update to brush up the links, EIPs, implementations (migrating the readiness matrix still pending) and EOF call links.

The design is far from ideal, but we can iterate on this, for now just fixing the obvious stuff.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,13 +105,27 @@ <h2>Implementations</h2>
...
-                  <li class="style3">
-                    <span class="icon solid fa-code-branch"></span>
-                    <strong>EOF Tests</strong>
-                    <div><a href="https://github.com/ethereum/tests/tree/develop/src/EOFTestsFiller">- Validation Tests</a></div>
-                    <div><a href="https://github.com/ethereum/tests/tree/develop/src/EIPTestsFiller/StateTests/stEOF">- Execution Tests</a></div>
-                    </ul>
-                  </li>
+									<li class="style4">
+										<span class="icon solid fa-code-branch"></span>
+                    <strong><a href="https://github.com/NethermindEth/nethermind/commits/feature/evm/eof">Nethermind 🚧</a></strong>
```
Comment (axic) [inline index.html:110]:
```suggestion
                    <strong><a href="https://github.com/NethermindEth/nethermind/pull/6896">Nethermind 🚧</a></strong>
```

File: index.html [hunk 2]
```diff
@@ -105,13 +105,27 @@ <h2>Implementations</h2>
...
+									<li class="style5">
+										<span class="icon solid fa-code-branch"></span>
+                    <strong><a href="https://github.com/ethereum/go-ethereum/pull/29518">Geth 🚧</a></strong>
+									</li>
+									<li class="style1">
+										<span class="icon solid fa-code-branch"></span>
+                    <strong><a href="https://github.com/ethereum/solidity/pull/13825">Solidity 🚧</a></strong>
+									</li>
+									</ul>
+									<ul class="statistics">
```
Comment (axic) [inline index.html:121]: Would link to ethereumjs too: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2453

Comment (pdobacz) [reply index.html:121]: oh yeah, it was all stale before, but after ACDE definitely need to include

Comment (axic) [reply index.html:121]: Alright can be done separately too

**Review by axic — APPROVED** [review]

**Pull request merged**